### PR TITLE
Redirect privacy policy to external source

### PR DESCRIPTION
**Pull request by madebyrogal**
DEL remove privacy policy page

## Description

Changes proposed in this pull request:

- Redirect privacy policy to external source

## Testing

Please test these to links:

image

## Related issue(s)

https://github.com/kubeshop/botkube-cloud/issues/874

### CODE REVIEW

File: src/pages/privacy/index.md [hunk 1]
```diff
@@ -1,146 +0,0 @@
...
-- Anonymous machine ID from [machineid](https://github.com/denisbrodbeck/machineid) library,
-- Executed command names, such as `login`, `install`, etc.
-
-#### How to opt out
-
-To disable sending the anonymous analytics, provide the `analytics.disable: true` override during Helm chart installation or upgrade. See the [Helm chart parameters](/configuration/helm-chart-parameters/#values) for more details about Helm chart configuration.
-
-To disable sending the anonymous analytics for Botkube CLI, execute the command `botkube telemetry disable`. This configuration will be stored locally in
-`~/.botkube/config.json` file, if this file is deleted, the telemetry will be enabled again.
-
```
Comment (pkosiec) [inline src/pages/privacy/index.md:51]: Where exactly the data is located in the https://botkube.io/privacy-policy? I think we're missing this.

**Review by pkosiec — APPROVED** [review]
LGTM from docs side but please don't merge it until we have the missing data on https://botkube.io/privacy-policy. Thanks!

File: src/pages/privacy/index.md (hunk 1 above)
Comment (madebyrogal) [reply src/pages/privacy/index.md:51]: @brampling PTAL

Comment (brampling) [reply src/pages/privacy/index.md:51]: @pkosiec Let's put this in the installation docs somewhere. It doesn't need to be in the privacy policy.

**Review by pkosiec — APPROVED** [review]

**Pull request merged**